docs/usermanual.md

```diff
@@ -197,7 +197,7 @@ This section specifies the topics and messages allowed when using MQTT as the tr
 the topics used with the MQTT protocol contain the same prefix:
 
 ```text
-<apiKey>/<deviceId>
+/json/<apiKey>/<deviceId>
```

Comment:
This is UL agent, this should be `/ul`...

Reply:
fixed in https://github.com/telefonicaid/iotagent-ul/pull/401/commits/e0e7a41dad19ca7007e51f26c0d01df77523edcf